go/tosca/utils.go

```diff
@@ -64,3 +66,10 @@ func GetStorageStatus(original, current, new Word) StorageStatus {
 	// Default
 	return StorageAssigned
 }
+
+func SizeInWords(size uint64) uint64 {
```

Comment:
Please add documentation for this function, as it is now at such a prominent place.